students/slupek_bartosz/lesson_03_functions/Calculator.py

```diff
@@ -0,0 +1,93 @@
+def multiply():
+    print("MULTIPLY")
+    print("Input 1st operand:")
```

Comment:
pobieranie danych od użytkownika można wydzielić w postaci osobnej funkcji i wykorzystać zwracanie wielu wartości/zmiennych `return a, b, c, d...`
np:
`def f1():`
`...`
`return a, b`
Użycie funkcji z natychmiastowym rozpakowaniem zmiennych:
`n1, n2 = f1()`

można również rozpakować je później i wyniki trzymać jako tupla:
`liczby = f1()`
`jakas_fun(liczby[0], liczby[1])`
ale to jest temat na później